src/azure-cli/azure/cli/command_modules/hdinsight/commands.py

```diff
@@ -4,7 +4,7 @@
 # --------------------------------------------------------------------------------------------
 
 
-def load_command_table(self, _):
+def load_command_table(self, _):  # pylint: disable=too-many-statements
```

Comment:
># pylint: disable=too-many-statements [](start = 34, length = 37)

Why bother to add the suppression comment?

Reply:
`azdev style hdinsight` returned an error indicated that too many statements under load_command_table

---
In reply to: [466331164](https://github.com/Azure/azure-cli/pull/14692#discussion_r466331164) [](ancestors = 466331164)